README.md

```diff
@@ -81,14 +80,14 @@ dolphinscheduler-dist/target/apache-dolphinscheduler-incubating-${latest.release
 
 ### Thanks
 
-Dolphin Scheduler is based on a lot of excellent open-source projects, such as google guava, guice, grpc, netty, ali bonecp, quartz, and many open source projects of apache and so on.
-Thanks to the contributions of these open source projects, the birth of Dolphin Scheduler comes true. We are very grateful for all the open source software used! We hope that we are not only the beneficiaries of open source, but also be the open source contributors. Besides, we also hope that partners who have the same passion and conviction for open source will join in and contribute to open source!
+Dolphin Scheduler is based on a lot of excellent open-source projects, such as google guava, guice, grpc, netty, ali bonecp, quartz, and many open-source projects of Apache and so on.
+Thanks to the contributions of these open-source projects, the dream of Dolphin Scheduler comes true. We are very grateful for all the open-source softwares used! We hope that we are not only the beneficiaries of open-source, but also be the open-source contributors. Besides, we expect the partners who have the same passion and conviction to open-source will join in and contribute to open-source community!
```

Comment:
Thanks to the contributions of these open-source projects, the dream of Dolphin Scheduler comes true.
Thanks to the contributions of these open-source projects make (or assist) the dream of Dolphin Scheduler comes true.

but also be the open-source contributors.
but also reward back to the community.(maybe better)